Description: Fix requirements.txt
 This avoids FTBFS as dpkg-gencontrol doesn't like the syntax patched here.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2016-09-17

--- designate-3.0.0~b3.orig/requirements.txt
+++ designate-3.0.0~b3/requirements.txt
@@ -3,13 +3,13 @@
 # process, which may cause wedges in the gate later.
 
 Babel>=2.3.4 # BSD
-eventlet!=0.18.3,>=0.18.2 # MIT
-Flask!=0.11,<1.0,>=0.10 # BSD
+eventlet
+Flask
 greenlet>=0.3.2 # MIT
 Jinja2>=2.8 # BSD License (3 clause)
-jsonschema!=2.5.0,<3.0.0,>=2.0.0 # MIT
+jsonschema
 keystoneauth1>=2.10.0 # Apache-2.0
-keystonemiddleware!=4.1.0,!=4.5.0,>=4.0.0 # Apache-2.0
+keystonemiddleware
 netaddr!=0.7.16,>=0.7.12 # BSD
 oslo.config>=3.14.0 # Apache-2.0
 oslo.concurrency>=3.8.0 # Apache-2.0
@@ -24,18 +24,17 @@ oslo.utils>=3.16.0 # Apache-2.0
 Paste # MIT
 PasteDeploy>=1.5.0 # MIT
 pbr>=1.6 # Apache-2.0
-pecan!=1.0.2,!=1.0.3,!=1.0.4,>=1.0.0 # BSD
+pecan
 python-designateclient>=1.5.0 # Apache-2.0
 python-neutronclient>=5.1.0 # Apache-2.0
-Routes!=2.0,!=2.1,!=2.3.0,>=1.12.3;python_version=='2.7' # MIT
-Routes!=2.0,!=2.3.0,>=1.12.3;python_version!='2.7' # MIT
+Routes
 requests>=2.10.0 # Apache-2.0
 six>=1.9.0 # MIT
 SQLAlchemy<1.1.0,>=1.0.10 # MIT
 sqlalchemy-migrate>=0.9.6 # Apache-2.0
 stevedore>=1.16.0 # Apache-2.0
 suds-jurko>=0.6 # LGPLv3+
-setuptools!=24.0.0,>=16.0 # PSF/ZPL
+setuptools
 WebOb>=1.2.3 # MIT
 dnspython>=1.14.0 # http://www.dnspython.org/LICENSE
 dnspython3>=1.12.0;python_version>='3.0' # http://www.dnspython.org/LICENSE
